packages/core/contracts/insured-bridge/BridgeAdmin.sol

```diff
@@ -116,6 +116,19 @@ contract BridgeAdmin is BridgeAdminInterface, Ownable, Lockable {
         emit SetDepositContracts(_chainId, _depositContract, _messengerContract);
     }
 
+    /**
+     * @notice Enables the current owner to transfer ownership of a set of owned bridge pools to a new owner.
+     * @dev Only callable by the current owner.
+     * @param _bridgePools array of bridge pools to transfer ownership.
+     * @param _newAdmin new admin contract to set ownership to.
+     */
+    function transferBridgePoolAdmin(address[] memory _bridgePools, address _newAdmin) public onlyOwner nonReentrant() {
+        for (uint8 i = 0; i < _bridgePools.length; i++) {
```

Comment:
What if we have more than 2^8 bridge pools? This seems potentially too tight, why not use uint16

Reply:
can just call the function multiple times but I'm happy to increase it to uint16

Reply:
Yeah my bad you can revert I missed that part